### DISCUSSION TITLE
Contact us form

### DISCUSSION THREAD

**Discussion by iTheMaverick** [twbs/bootstrap#39440] [Q&A]
I'm trying to make a contact us form, how do I make the information input go to an email?

**Comment by tommy6101** [comment]
After using HTML you will need PHP

**Comment by tommy6101** [comment]
```html
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Contact Us Form</title>
</head>
<body>
    <h2>Contact Us</h2>
    <form action="process_form.php" method="post">
        <label for="name">Name:</label>
        <input type="text" id="name" name="name" required><br>
        
        <label for="email">Email:</label>
        <input type="email" id="email" name="email" required><br>
        
        <label for="message">Message:</label>
        <textarea id="message" name="message" required></textarea><br>
        
        <input type="submit" value="Submit">
    </form>
</body>
</html>
```

**Reply by moshetanzer** [reply]
Hi Tommy,

Happy to help you. 

You need a php script to send email. Supposing you hosting your website on shared hosting?
Let me know if you need help.

**Comment by tommy6101** [comment]
<?php
if ($_SERVER["REQUEST_METHOD"] == "POST") {
    // Get form data
    $name = $_POST["name"];
    $email = $_POST["email"];
    $message = $_POST["message"];

    // Set the recipient email address
    $to = "your_email@example.com";

    // Set the subject of the email
    $subject = "New Contact Form Submission";

    // Build the email message
    $email_message = "Name: $name\n";
    $email_message .= "Email: $email\n";
    $email_message .= "Message:\n$message";

    // Send the email
    mail($to, $subject, $email_message);

    // Redirect back to the form with a success message
    header("Location: index.html?status=success");
} else {
    // Redirect back to the form with an error message
    header("Location: index.html?status=error");
}
?>
